pavelib/bok_choy.py

```diff
@@ -19,6 +19,7 @@
 BOKCHOY_OPTS = [
     ('test_spec=', 't', 'Specific test to run'),
     ('fasttest', 'a', 'Skip some setup'),
+    ('skip_clean', 'C', 'Skip cleaning repository before running tests'),
```

Comment:
ah, I see. Seems like the best thing is to keep it C then. thx for the explanation.